test/token-pool-proxy.test.js

```diff
@@ -53,4 +62,51 @@ contract('Token Pool', accounts => {
       expect(tokenAddr).to.be.equal(ctx.contracts.token.address)
     })
   })
+  describe('change manufacturer wallet', () => {
+
+    it("new address cannot be 0x0", async () => {
+
+      const fn = ctx.contracts.pool.changeManufacturerWallet.call(0x0, {from: ctx.actors.mfg})
+      await errors.expectRevert(fn)
+    })
+    
+    it("must be a manufacturer", async () => {
+
+      await ctx.contracts.member.addNetworkMember(ctx.actors.admin, false, false, false, 'test_pools', {from: ctx.actors.owner})
```

Comment:
can just use `ctx.actors.alice` to avoid an additional call in the test execution